test.sh

```diff
@@ -8,14 +8,16 @@ cd /usr/local/ecoservice
 godep go test eco/*
 
 cd /usr/local/otm/app/
-grunt check
+yarn run check
 
-export DISPLAY=":99.0"
-Xvfb $DISPLAY 2>/dev/null >/dev/null &
-testem ci
+yarn run test
```

Comment:
This was failing for me

```
$ testem ci
not ok 1 Firefox - Browser "firefox -no-remote -profile /tmp/testem.firefox http://localhost:7357/5143" exited unexpectedly.

1..1
# tests 1
# pass  0
# fail  1
error Command failed with exit code 1.
```

Changing the line to `xvfb-run yarn run test` worked.

Reply:
Fixed in https://github.com/OpenTreeMap/otm-vagrant/pull/54/commits/3fc3ef0feb6b66dcf8c68e33d660b30deeaea2aa